MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+title: Data annotations in Maui DataForm control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in Maui DataForm (SfDataForm)
```

Comment:
# Data annotations in .NET MAUI DataForm (SfDataForm)

Reply:
Title changed